### PR TITLE
mpls: basic LDP configuration

### PR DESCRIPTION
**Pull request by currite**
Basic configuration to act as a P router in an MPLS network.

### CODE REVIEW

File: docs/routing/index.rst [hunk 1]
```diff
@@ -8,13 +8,23 @@ Routing
...
    pbr
    rip
-   static
-   bfd
-   mss-clamp
    routing-policy
-   multicast
+   rpki
+   static
+
```
Comment (c-po) [inline docs/routing/index.rst:22]: Why so many empty lines?

Comment (currite) [reply docs/routing/index.rst]: I tried to sort them out alphabetically.
Where are there empty lines?

Comment (currite) [reply docs/routing/index.rst]: Do you mean here in docs/routing/index.rst?

Comment (currite) [reply docs/routing/index.rst]: Found the empty lines! I updated it.

**Pull request merged**